crucible-llvm/crucible-llvm.cabal

```diff
@@ -1,6 +1,6 @@
 Cabal-version: 2.2
 Name:          crucible-llvm
-Version:       0.6.0.99
+Version:       0.9.0
```

Comment:
The convention up until this point in the `crucible` repo has been to use `0.9` instead of `0.9.0`. While I don't have a strong opinion on which is better, we should probably be consistent here (and in the other `.cabal` files).